spec/lib/doi_search_spec.rb

```diff
@@ -1,36 +1,101 @@
-
 describe DoiSearch do
+  let(:other_doi) { '10.1038/psp.2013.66' }
   let(:doi_value) { '10.1016/j.mcn.2012.03.007' }
-  let(:doi_identifier) do
-    FactoryBot.create(:publication_identifier,
-                       identifier_type: 'doi',
-                       identifier_value: doi_value,
-                       identifier_uri: "http://dx.doi.org/#{doi_value}")
-  end
+  let(:doi_identifier) { create(:doi_pub_id, identifier_value: doi_value) }
+  let(:wos_html) { File.read('spec/fixtures/wos_client/medline_encoded_records.html') }
+  let(:wos_records) { WebOfScience::Records.new(encoded_records: wos_html) }
 
   before(:each) do
     doi_identifier.publication.save
+    allow(Settings.SCIENCEWIRE).to receive(:enabled).and_return(false) # default
+    allow(Settings.WOS).to receive(:enabled).and_return(false) # default
   end
 
   describe '.search' do
-    it 'returns one document ' do
-      VCR.use_cassette('doi_search_spec_one_doc') do
-        result = DoiSearch.search doi_value
+    subject(:result) { described_class.search(doi_value) }
 
-        expect(result.size).to eq 1
-        expect(result.first[:sw_id]).to eq '60813767'
+    shared_examples 'sciencewire one hit' do
+      it 'returns Array of one document (Hash)' do
+        expect(result).to match [a_hash_including(sw_id: '60813767', provenance: 'sciencewire')]
       end
     end
 
-    it 'queries sciencewire if the locally found pub is non-sciencewire' do
-      VCR.use_cassette('doi_search_manual_doi_local') do
-        doi_identifier.publication.pub_hash.update(provenance: 'cap')
-        doi_identifier.publication.save
-        expect(ScienceWireClient).to receive(:new).and_call_original
-        result = DoiSearch.search doi_value
-        expect(result.size).to eq 1
-        expect(result.first[:provenance]).to eq 'sciencewire'
+    context 'ScienceWire and WOS both disabled' do
+      it 'only searches locally, returning Array of one Publication' do
+        expect(ScienceWireClient).not_to receive(:new)
+        expect(WebOfScience).not_to receive(:queries)
+        expect(described_class.search(doi_value)).to eq([doi_identifier.publication])
```

Comment:
`described_class.search(doi_value)` -> `result` ?

Reply:
`.search` returns an array.  (In this test, an array w/ 1 element)

Reply:
- ln 15:  `subject(:result) { described_class.search(doi_value) }`
- ln 27:  `described_class.search(doi_value)`

So, looks like ln 27 can be:
- `expect(result).to eq([doi_identifier.publication])`